pgcommitfest/commitfest/views.py

```diff
@@ -531,6 +533,7 @@ def patchlist(request, cf, personalized=False):
     )
 
 
+@transaction.atomic  # tie the patchlist() query to Tag.objects.all()
```

Comment:
I'd be inclined to define Tag as an append/update-only table - soft deletes only.  Think PostgreSQL enums in recommended usage (but with a soft delete capability).  No real point enforcing a transaction to read its contents in that case.  Even if we got an overly fresh read the attributes of Tag that would be valid data.

Reply:
> Is there some hidden cost to `@transaction.atomic` that's not in the documentation, that I should be aware of?

Not that I know of.  The goal of using soft deletes still stands.  Don't show "hidden/deleted" tags in the UI but they are still there and can be re-enabled.  With name/description separated from label so long as the purpose is deemed valid the label can be changed to better match community expectations - which I suspect would happen if we removed and then reinstated a tag.  It doesn't have to a hard-and-fast rule; especially if we don't functionally depend on it.